docs/getting-started/install-zowe-sdks.md

```diff
@@ -0,0 +1,87 @@
+# Zowe SDKs installation
+
+Leverage the Zowe Client Software Development Kits (SDKs) to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Client Node.js SDK
+- Zowe Client Python SDK *technical preview*
+- Zowe Client Kotlin SDK *technical preview*
+- Zowe Client Java SDK *under development*
+
+## SDK documentation
+
+For detailed SDK documentation, see the following:
+- [Zowe Node.js SDK](https://docs.zowe.org/stable/typedoc/index.html)
+- [Zowe Client Python SDK](https://zowe-client-python-sdk.readthedocs.io/en/latest/)
+
+## Software requirements
+
+### Node.js
+
+If you install Node SDK packages from the online registry, the required dependencies are installed automatically.
+
+If you download Node SDK packages from Zowe.org, the folder contains dependencies that you must install manually. Extract the `TGZ` files from the folder, copy the files to your project, and issue the following commands to install the dependencies.
+
+```
+npm install imperative.tgz
+```
+
+```
+npm install core-for-zowe-sdk.tgz
+```
+
+### Python *technical preview*
+
+If you install Python SDK packages from the online registry, the required dependencies are installed automatically.
+
+If you download the Python SDK packages from Zowe.org, the downloaded folder contains dependencies that you must install manually. Extract the `WHL` files from the folder, copy the files to your project, and issue the following command for each dependency:
+
+```
+pip install <fileName>.whl
+```
+
+### Kotlin *technical preview*
+
+To install this library in your project, use a build tool such as Maven, Gradle or Ant. Use the following link to get necessary artifacts: https://zowe.jfrog.io/artifactory/libs-release/org/zowe/sdk/zowe-kotlin-sdk/.
+
+To set up the SDK as a dependency:
```

Comment:
@KUGDev -- would this work for only Maven? and where would these changes be made?

Reply:
For Gradle (Kotlin):
`implementation("org.zowe.sdk:zowe-kotlin-sdk:$version")`

Reply:
Yes, my previous line is for those, who prefer to use Gradle

Reply:
@KUGDev -- In the Java section, we list the specific file that needs to be updated for Maven and Gradle projects. Also, the Java Gradle code snippet seems to include more information (name and version).

For Kotlin, should the Maven and Gradle information include file names too? And should the Kotlin Gradle code snipped include name and version? 

I would like all the information for the different sdks to be as consistent as possible.
<img width="616" alt="image" src="https://github.com/zowe/docs-site/assets/109533923/0fa0b371-7e5f-48c2-b460-d3c6d9462e5d">